.github/actions/repo-gardening/src/index.js

```diff
@@ -31,6 +32,10 @@ const automations = [
 		action: [ 'opened', 'reopened', 'synchronize', 'edited', 'labeled' ],
 		task: ifNotClosed( addLabels ),
 	},
+	{
+		event: 'push',
```

Comment:
Would it make more sense to do
```suggestion
		event: 'pull_request',
		action: [ 'closed' ],
```
instead of trying to parse merges to master? That way it'll also clean up merges to feature branches and such too.

(that would, of course, require modification of the task too to get the PR number more directly)

Reply:
Yes you're right, it does simplify things quite a bit. I've updated the PR accordingly.